man/git-generate-changelog.1

```diff
@@ -13,115 +13,115 @@
 Automatically generate changelog from your tags, issues, labels and pull requests on GitHub\.
 .
 .SH "OPTIONS"
-\-u, \-\-user [USER]
+\-u, \-\-user USER
 .
 .P
 Username of the owner of target GitHub repo\.
 .
 .P
-\-p, \-\-project [PROJECT]
+\-p, \-\-project PROJECT
 .
 .P
 Name of project on GitHub\.
 .
 .P
-\-t, \-\-token [TOKEN]
+\-t, \-\-token TOKEN
 .
 .P
 To make more than 50 requests per hour your GitHub token is required\. You can generate it at: https://github\.com/settings/tokens/new
 .
 .P
-\-f, \-\-date\-format [FORMAT]
+\-f, \-\-date\-format FORMAT
 .
 .P
 Date format\. Default is %Y\-%m\-%d\.
 .
 .P
-\-o, \-\-output [NAME]
+\-o, \-\-output NAME
 .
 .P
 Output file\. To print to STDOUT instead, use blank as path\. Default is CHANGELOG\.md\.
 .
 .P
-\-b, \-\-base [NAME]
+\-b, \-\-base NAME
 .
 .P
-Optional base file to append generated changes to\.
+Optional base file to append generated changes to\. Default is HISTORY\.md\.
 .
 .P
-\-\-summary\-label [LABEL]
+\-\-summary\-label LABEL
 .
 .P
 Set up custom label for the release summary section\. Default is ""\.
 .
 .P
-\-\-breaking\-label [LABEL]
+\-\-breaking\-label LABEL
 .
 .P
 Set up custom label for breaking changes section\. Default is "\fBBreaking changes:\fR"\.
 .
 .P
-\-\-enhancement\-label [LABEL]
+\-\-enhancement\-label LABEL
 .
 .P
 Set up custom label for enhancements section\. Default is "\fBImplemented enhancements:\fR"\.
 .
 .P
-\-\-bugs\-label [LABEL]
+\-\-bugs\-label LABEL
 .
 .P
 Set up custom label for bug\-fixes section\. Default is "\fBFixed bugs:\fR"\.
 .
 .P
-\-\-deprecated\-label [LABEL]
+\-\-deprecated\-label LABEL
 .
 .P
 Set up custom label for deprecated section\. Default is "\fBDeprecated:\fR"\.
 .
 .P
-\-\-removed\-label [LABEL]
+\-\-removed\-label LABEL
 .
 .P
 Set up custom label for removed section\. Default is "\fBRemoved:\fR"\.
 .
 .P
-\-\-security\-label [LABEL]
+\-\-security\-label LABEL
 .
 .P
 Set up custom label for security section\. Default is "\fBSecurity:\fR"\.
 .
 .P
-\-\-issues\-label [LABEL]
+\-\-issues\-label LABEL
 .
 .P
 Set up custom label for closed\-issues section\. Default is "\fBClosed issues:\fR"\.
 .
 .P
-\-\-header\-label [LABEL]
+\-\-header\-label LABEL
 .
 .P
 Set up custom header label\. Default is "# Changelog"\.
 .
 .P
-\-\-configure\-sections [HASH, STRING]
+\-\-configure\-sections HASH, STRING
 .
 .P
 Define your own set of sections which overrides all default sections\.
 .
 .P
-\-\-add\-sections [HASH, STRING]
+\-\-add\-sections HASH, STRING
```

Comment:
Maybe we should investigate this. Me, reading it, can't from the example figure out real quick what I can write here. Is Hash defined anywhere? How do I tell the CLI to enter a hash here? Can I do that in the CLI version?

I understand that this was not introduced by your amazing changes, but perhaps we need to capture this in an Issue.